src/Fixer/Basic/CurlyBracesPositionFixer.php

```diff
@@ -36,6 +36,11 @@ final class CurlyBracesPositionFixer extends AbstractFixer implements Configurab
 {
```

Comment:
btw, rule was renamed, if anything, shall go to Fixer under new name, ref https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7334